docs/development/development.md

```diff
@@ -90,3 +90,39 @@ $ make run PROFILE=gke
 ```
 
 To stop execution and cleanup the deployment, hit `Ctrl+C`.
+
+## Testing
+
+`nats-operator` includes an end-to-end test suite that is used to validate the implementation.
+As the test suite depends on having certain, advanced features enabled on the target Kubernetes cluster, Minikube is currently the only supported environment for running the test suite.
+To launch a Minikube cluster suitable for running the end-to-end test suite, you may run:
+
+```console
+$ minikube start \
+    --extra-config=apiserver.service-account-signing-key-file=/var/lib/minikube/certs/apiserver.key \
+    --extra-config=apiserver.service-account-issuer=api \
+    --extra-config=apiserver.service-account-api-audiences=api \
+    --extra-config=apiserver.service-account-key-file=/var/lib/minikube/certs/sa.pub \
```

Comment:
so the path in minikube changed for these in the latest version, I see now

Reply:
We also needed to update the Ubuntu version.